model/src/main/kotlin/Mappers.kt

```diff
@@ -54,7 +40,7 @@ private val mapperConfig: ObjectMapper.() -> Unit = {
     registerModule(JavaTimeModule())
     disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
 
-    registerModule(ortModelModule)
+    registerModule(SimpleModule("OrtModelModule"))
```

Comment:
This should not be required anymore if the module is empty.

Reply:
Even better, thanks! Removed to verify.